gensim/models/doc2vec.py

```diff
@@ -494,7 +494,17 @@ def n_similarity(self, ds1, ds2):
         v2 = [self[doc] for doc in ds2]
         return dot(matutils.unitvec(array(v1).mean(axis=0)), matutils.unitvec(array(v2).mean(axis=0)))
 
+    def similarity_unseen_docs(self, model, doc_words1, doc_words2, alpha=0.1, min_alpha=0.0001, steps=5):
+        """
+        Compute cosine similarity between two post-bulk out of training documents.
 
+        Document should be a list of (word) tokens.
+        """
+        d1 = model.infer_vector(doc_words1, alpha, min_alpha, steps)
```

Comment:
Use named parameters (rather than positional).

It's safer in case the argument order inside `infer_vector` changes in the future.